recipe/CMakeLists.txt

```diff
@@ -70,10 +72,18 @@ file(GLOB giflib_MAN doc/*.1)
 ### Build library / tools
 
 add_library(gif SHARED ${giflib_SRC})
-target_link_libraries(gif m)
+
+if (NOT WIN32)
+    target_link_libraries(gif m)
+endif()
+
 set_target_properties(gif PROPERTIES VERSION ${LIBVER} SOVERSION ${LIBMAJOR})
 if(WIN32)
-    set_target_properties(gif PROPERTIES RUNTIME_OUTPUT_NAME "gif-${LIBMAJOR}")
```

Comment:
Why remove this?

Reply:
had problems linking, not sure if it was related to this in the end. let's see...